### PR TITLE
PHP target details

### PR DESCRIPTION
**Pull request by markknol**
Contains; Getting started with PHP.

### CODE REVIEW

File: 12-target-details.tex [hunk 1]
```diff
@@ -301,6 +301,46 @@ \section{Neko}
...
+\begin{lstlisting}
+-php bin
+-main Main
+\end{lstlisting}
+
+The compiler outputs in the given \emph{bin}-folder, which contains the generated PHP classes that prints the traced message when you run it.
+
+\paragraph{Run the PHP}
+
+To run the PHP a web server is needed, such as \href{http://httpd.apache.org/}{Apache}. An easy-to-install Apache distribution for local development is \href{https://www.apachefriends.org/download.html}{XAMPP}.
```
Comment (Simn) [inline 12-target-details.tex]: That's overkill, all you need is a PHP interpreter. You can easily run PHP from command line (I do it all the time when testing).

File: 12-target-details.tex [hunk 2]
```diff
@@ -301,6 +301,46 @@ \section{Neko}
 
 \section{PHP}
 \label{target-php}
+\state{NoContent}
+
+\subsection{Getting started with PHP}
+\label{target-php-getting-started}
+
+Haxe can be a powerful tool for developing PHP applications. Let's look at our first example showing the toolchain.
```
Comment (Simn) [inline 12-target-details.tex]: This doesn't carry any value whatsoever.

File: 12-target-details.tex [hunk 3]
```diff
@@ -301,6 +301,46 @@ \section{Neko}
...
 \section{PHP}
 \label{target-php}
+\state{NoContent}
+
+\subsection{Getting started with PHP}
+\label{target-php-getting-started}
+
+Haxe can be a powerful tool for developing PHP applications. Let's look at our first example showing the toolchain. 
+
+Create a new folder and save this class as \ic{Main.hx}.
```
Comment (Simn) [inline 12-target-details.tex]: I'd remove the line above and start here with "To get started with Haxe/PHP, create a new folder..."

We might want to label the section "Getting started with Haxe/PHP" too (and do the same for other targets) to not confuse google.

File: 12-target-details.tex [hunk 4]
```diff
@@ -301,6 +301,46 @@ \section{Neko}
...
+
+\haxe{assets/HelloPHP.hx}
+
+To compile, either run this from the command line:
+
+\begin{lstlisting}
+haxe -php bin -main Main
+\end{lstlisting}
+
+.. or create and run (double-click) a file called \ic{compile.hxml}. In this example the hxml-file should be in the same directory as the example class.
```
Comment (Simn) [inline 12-target-details.tex]: Ellipsis in technical documents are not good, but having only two dots is even worse. ;)

Can we restructure this?

Comment (markknol) [inline 12-target-details.tex]: It is also used in the flash/javascript section, ill have to fix those too.

**Pull request merged**